modules/ldap_groups/lib/open_project/ldap_groups/patches/user_patch.rb

```diff
@@ -0,0 +1,13 @@
+module OpenProject::LdapGroups
+  module Patches
+    module UserPatch
+      def self.included(base) # :nodoc:
+        base.class_eval do
+          has_many :ldap_groups_memberships,
+                   class_name: '::LdapGroups::Membership',
+                   dependent: :destroy
```

Comment:
Not relevant here as it is already carried out asynchronously but in other scenarios we could make use of `destroy_async`(https://bigbinary.com/blog/rails-6-1-allows-associations-to-support-destroy_async-option-with-dependent-key)

Reply:
Good to know, thanks!